### ISSUE TITLE
Latest CasAuthenticationFilter uses different default security endpoint (/login/cas)

### ISSUE THREAD

**Issue by cm325** [jgribonvald/demo-spring-cas-angular#4]
I just worked through using your modifications for a new jhipster build (they were a lifesaver!!), and it wasn't working, and the problem was that the lastest spring security CasAuthenticationFilter has a different security endpoint that gets registered by default, so instead of:

/j_spring_cas_security_check

its now:

/login/cas

With so many changes coming from Spring 3, xml config, etc, it would be hard for a new user to catch this-

**Comment by jgribonvald** [comment]
Thanks for your feed back, you are right about that, an upgrade is needed
; )

**Comment by cm325** [comment]
No problem. We've really appreciated your jhipster/cas demo, there is a real lack of documentation at the moment it seems on getting spring security configuring correctly with Java config, and of course CAS. 

I'm curious, did you also look at the possibility of casifying the oauth version of jhipster, now that cas 4 has full oauth support?

**Comment by jgribonvald** [comment]
No, sorry not again... I planned to migrate our CAS from a version 3.4.12
to 4.x at the end of the year only. We wait that all feature for failover
in CAS were fonctionnal before to explore...
